hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CachePutAllBackupOperation.java

```diff
@@ -78,6 +80,16 @@ public void run() throws Exception {
     }
 
     @Override
+    public ObjectNamespace getServiceNamespace() {
```

Comment:
may be not subject of this pr but it would be better to put this method to a single place to eliminate code duplications